crates/workspace/src/workspace_settings.rs

```diff
@@ -14,6 +14,7 @@ pub struct WorkspaceSettings {
     pub restore_on_startup: RestoreOnStartupBehaviour,
     pub drop_target_size: f32,
     pub when_closing_with_no_tabs: CloseWindowWhenNoItems,
+    pub download_missing_binaries: bool,
```

Comment:
Can't this just use the language settings `DownloadConfiguration` struct? You can make the struct transparent by annotating it with `#[serde(transparent)]` and then it can behave literally like a boolean.

Reply:
This code is WIP, thanks for the feedback